index.html

```diff
@@ -22,17 +22,26 @@ <h2 class="card__question">
               In the Kingdom Heart series who provides the english voice for
               Master Eraqus?
             </h2>
-            <button class="card__button-answer" type="button">
+            <button
+              class="card__button-answer"
+              type="button"
+              data-answer-id="1"
+            >
               Show answer
             </button>
-            <p class="card__answer">Mark Hamill</p>
+            <p class="card__answer.hidden" data-answer-id="1">Mark Hamill</p>
```

Comment:
in der class den punkt weg und ein leerzeichen rein

Reply:
Wenn ich das mache, funktioniert der Toggle nicht mehr :(

Reply:
Hab's gelöst :)